styles.css

```diff
@@ -225,7 +268,8 @@ section.work_section ul li > span {
 div.canopy {
   display: flex;
   flex-direction: row;
-  justify-content: baseline;
+
+  /* justify-content: baseline; */
```

Comment:
- [ Required ] I would like to suggest that you delete this commented line of code, this is because commented code makes a project look unprofessional and is widely considered as a sign of code that is not clean. Please rectify this.